### DISCUSSION TITLE
User Auditor Troubleshooting/help

### DISCUSSION THREAD

**Discussion by bradjohnson1102** [utmstack/UTMStack#856] [Q&A] [answered]
I have roughly 15,000 users in my organization.  The user auditor tool doesn't appear to load all users.  It stops somewhere in the "A" area, meaning I cannot find information for anyone whose name does not begin with the letter A.  Please advise.

**Comment by c3s4rfred** [comment, marked as answer]
Hi @bradjohnson1102 the user auditor actually doesn't load all users after scanning logs, these are the event_id that we scanned to load users from 
![image](https://github.com/user-attachments/assets/6092b1d9-092a-44a1-9d76-9a0e8fed288a)

Best regards
